challenges/edabit-arrays.js

```diff
@@ -3,14 +3,44 @@
 // 1. Create a function that takes an array of positive and negative numbers. Return an array where the first element is the count of positive numbers and the second element is the sum of negative numbers.
 // https://edabit.com/challenge/xXJLZry3vYd4erPct
 
+function countPosSumNeg(nums) {
+  if (nums === [] || nums === null) {
+    return []
+  }
+  let positiveCount = nums.reduce((acc, next) => {
+    if (next > 0) {
+      return acc += 1
+    } else {
+      return acc
+    }
+  }, 0)
+
+  let negativeSum = nums.reduce((acc, next) => {
+    if (next < 0) {
+      return acc + next
+    } else {
+      return acc
+    }
+  }, 0)
```

Comment:
I feel like you should be able to combine these two into one call.  Perhaps the accumulator is  an array with two things, so you increment acc[0] on positive and acc[1] on negative.